bin/create-readme-for-blog-utils

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+
+const rootDir = path.join(__dirname, '..');
+const distDir = path.join(rootDir, 'dist');
+
+const content = `# addons-frontend-blog-utils
+
+Please see: https://github.com/mozilla/addons-frontend#addons-frontend-blog-utils
+`;
+
+fs.writeFile(
+  path.join(distDir, 'README.md'),
```

Comment:
nit: this is a little overwritten - we only have a single file so we could have a single `path.join`